### PR TITLE
A reddit models copy

### PR DESCRIPTION
**Pull request by iliebabcenco**
In this project, we created:
1. User, Post, Comment models;
2. Associations between User (has_many) and Posts (belongs_to);
3. Associations between User (has_many) and Comments (belongs_to);
4. Associations between Post (has_many) and Comments (belongs_to).

### CODE REVIEW

**Comment by mimipeshy** [comment]
## STATUS: CHANGES REQUESTED 🔴 ♻️ 
Hello team @iliebabcenco and @felix-vondee 👋🏾 👋🏾 , you have made such great progress in this milestone 🥇 . Your project works as expected. However the following change is needed before approval

- [x] In your README file add instructions on how to set up your project.

That's all 😄, kindly submit another review when done
If you need any clarification, kindly post your comment here and tag me @mimipeshy and will be glad to help

Regards,
Peris Ndanu

**Review by BrianSammit — APPROVED** [review]
STATUS APPROVED :green_circle: :clap: 
--------------
Hello @iliebabcenco @felix-vondee great job working on the required changes :muscle: :smile: 
-------------
Now you can merge this branch :+1: :100: 
HAPPY CODNG!! GOOD LUCK WITH THE NEXT ONES :crossed_fingers: :four_leaf_clover: 

tenor

**Pull request merged**
